src/legacy/core_plugins/kibana/public/discover/get_inner_angular.ts

```diff
@@ -153,9 +148,8 @@ export function initializeInnerAngularModule(
       'discoverConfig',
       'discoverI18n',
       'discoverPrivate',
+      'discoverPersistedState',
```

Comment:
dear @alexwizp, got a problem here, you've removed it recently, which is great. but I've to add it in my PR, else I get the following screen:

![Bildschirmfoto 2020-03-02 um 11 10 27](https://user-images.githubusercontent.com/463851/75666785-bae9c400-5c76-11ea-9b24-6927999e0d2d.png)

Do you have an idea why?

Reply:
let me see, please don't merge it. We should find reason

Reply:
resolved